core/consensus/common/Cargo.toml

```diff
@@ -7,7 +7,8 @@ edition = "2018"
 
 [dependencies]
 derive_more = "0.14.0"
-libp2p = { version = "0.10.0", default-features = false }
+# libp2p = { version = "0.10.0", default-features = false }
+libp2p = { path = "../../../../rust-libp2p", version = "0.10.0", default-features = false }
```

Comment:
This is irrelevant for the PR, but for what it's worth, you can use a `[patch]` section in the root Cargo.toml to ask cargo to use your custom version.

Reply:
:+1:  Thanks for the reminder. Now that you mention it, I actually used that once before but somehow forgot about it. Memory like a goldfish ...